great_expectations/experimental/datasources/data_asset/data_connector/file_path_data_connector.py

```diff
@@ -38,6 +36,61 @@
 logger = logging.getLogger(__name__)
 
 
+def data_references_loader(
```

Comment:
please encapsulate this in the _get_data_connector implementation, we shouldn't be using a decorator